var/spack/repos/builtin/packages/relax/package.py

```diff
@@ -17,7 +17,7 @@ class Relax(CMakePackage):
 
     # The RELAX project stopped using a fixed release model since it moved to
     # ROOT 6, so any package checksum is a lie...
-    version('root6')
+    version('root6', sha256='1d24b1a0884bbe99d60f7d02fea45d59695c158ab5e53516ac3fb780eb460bb4')
```

Comment:
Ugh, what does the comment above mean?

Reply:
I don't know, but the package seemingly installs fine with the checksum added. @HadrienG2 would you mind elaborating what you meant with that comment?

Reply:
The "root6 release" of RELAX is not actually a release in the way Spack means it, it's more like a git branch : it can get updated over time, so the package content changes, and thus the checksum goes stale. This results in a checksum mismatch which will be misinterpreted as package corruption or symptom of a security breach, when it's just normal evolution of a perma-unstable codebase...

...or at least that was my evaluation of the situation at the time where I wrote this comment. Judging from the package timestamp on the LCG mirror, it hasn't been updated in so long that it's probably fair to say that development has now fully stopped, making the root6 branch stable, which means that now it is probably safe enough to add a checksum and delete the comment.

Reply:
Thanks, @HadrienG2 ! That explains it. And yes, seems like nothing happened with the package version since that time, so I removed the comment.